twist_mux:
  ros__parameters:
    topics:

      keybaord:
        name    : keyboard
        topic   : cmd_vel_key
        timeout : 0.5
        priority: 100

      joystick:
        name    : joystick
        topic   : cmd_vel_joy
        timeout : 0.5
        priority: 80

      hand_gesture:
        name    : hand_gesture
        topic   : cmd_vel_gesture
        timeout : 0.5
        priority: 60

      navigation:
        name    : navigation
        topic   : cmd_vel_nav
        timeout : 0.5
        priority: 40

    locks:
      emergency_stop:
        name    : emergency_stop
        topic   : emergency_stop
        timeout : 0.0
        priority: 255